virttest/qemu_vm.py

```diff
@@ -4860,6 +4860,12 @@ def _set_migrate_capability(vm, capability, value, is_src_vm=True):
                         if os.path.isfile(save2):
                             os.remove(save2)
 
+            # FIXME: Never remove the source VM's files when doing a local
+            # migration because these files are still being used by the
+            # target VM, let the target VM do the cleanup work
```

Comment:
I think there is a situation in which the target VM still can not clean up the `temporary_image_snapshots` for migrating the VM in the `exec` way.
Since this way, the clone will not do the `create` action so the `temporary_image_snapshots` is still empty before being destroyed.

Reply:
> I think there is a situation in which the target VM still can not clean up the `temporary_image_snapshots` for migrating the VM in the `exec` way. Since this way, the clone will not do the `create` action so the `temporary_image_snapshots` is still empty before being destroyed.

Hi @zhencliu 
My bad, it will still be created.